cli/src/commands/explore/calls.rs

```diff
@@ -0,0 +1,145 @@
+    let Some(call) = calls_enum_type_def.variants.iter().find(|variant| variant.name.to_lowercase() == call_name.to_lowercase()) else {
+        let available_calls = print_available_calls(calls_enum_type_def, pallet_name);
+        let description = format!("Usage:\n    subxt explore {pallet_name} calls <CALL>\n        explore a specific call within this pallet\n\n{available_calls}", );
+        return Err(eyre!("\"{call_name}\" call not found in \"{pallet_name}\" pallet!\n\n{description}"));
+    };
+
+    // collect all the trailing arguments into a single string that is later into a scale_value::Value
+    let trailing_args = command.trailing_args.join(" ");
+
+    // if no trailing arguments specified show user the expected type of arguments with examples:
+    if trailing_args.is_empty() {
+        let mut type_description =
+            print_type_description(&call.fields, &metadata.runtime_metadata().types)?;
+        type_description = with_indent(type_description, 4);
+        let mut type_examples = print_type_examples(
+            &call.fields,
+            &metadata.runtime_metadata().types,
+            "SCALE_VALUE",
+        )?;
+        type_examples = with_indent(type_examples, 4);
+        let mut output = String::new();
+        write!(output, "Usage:\n    subxt explore {pallet_name} calls {call_name} <SCALE_VALUE>\n        construct the call by providing a valid argument\n\n")?;
+        write!(
+            output,
+            "The call expect expects a <SCALE_VALUE> with this shape:\n{type_description}\n\n{}\n\nYou may need to surround the value in single quotes when providing it as an argument."
+            , &type_examples[4..])?;
+        println!("{output}");
+        return Ok(());
+    }
+
+    // parse scale_value from trailing arguments and try to create an unsigned extrinsic with it:
+    let value = scale_value::stringify::from_str(&trailing_args).0.map_err(|err| eyre!("scale_value::stringify::from_str led to a ParseError.\n\ntried parsing: \"{}\"\n\n{}", trailing_args, err))?;
+    let value_as_composite = value_into_composite(value);
+    let offline_client = new_offline_client(metadata.clone());
+    let payload = tx::dynamic(pallet_name, call_name, value_as_composite);
+    let unsigned_extrinsic = offline_client.tx().create_unsigned(&payload)?;
+    let hex_bytes = format!("0x{}", hex::encode(unsigned_extrinsic.encoded()));
+    println!("Encoded call data:\n    {hex_bytes}");
+
+    Ok(())
+}
+
+fn print_available_calls(pallet_calls: &TypeDefVariant<PortableForm>, pallet_name: &str) -> String {
+    if pallet_calls.variants.is_empty() {
+        return format!("No <CALL>'s available in the \"{pallet_name}\" pallet.");
+    }
+    let mut output = format!("Available <CALL>'s in the \"{pallet_name}\" pallet:");
+
+    let mut strings: Vec<_> = pallet_calls.variants.iter().map(|c| &c.name).collect();
+    strings.sort();
+    for variant in strings {
+        output.push_str("\n    {");
```

Comment:
```suggestion
        output.push_str("\n    ");
```

Reply:
Ooops, you are right this `{` should not be there.